tools/gyp/docs/Hacking.md

```diff
@@ -34,7 +34,7 @@ See [Testing](Testing.md) for more details on the test framework.
 Note that it can be handy to look at the project files output by the tests
 to diagnose problems. The easiest way to do that is by kindly asking the
 test driver to leave the temporary directories it creates in-place.
-This is done by setting the enviroment variable "PRESERVE", e.g.
+This is done by setting the environment variable "PRESERVE", e.g.
```

Comment:
Changes to `gyp` should be upstreamed to `gyp-next`

Reply:
Done! Gave it the full treatment... https://github.com/nodejs/gyp-next/pull/267

Reply:
Great! Can you undo the changes to gyp in this PR? They will be updated by @nodejs-github-bot.